jest.config.js

```diff
@@ -1,10 +1,13 @@
 export default {
     testEnvironment: "jsdom",
     preset: "ts-jest",
-    globals: {
-        "ts-jest": {
-            diagnostics: false
-        }
+    transform: {
+        "'^.+\\.ts$'": [
+            "ts-jest",
+            {
+                diagnostics: false
+            }
+        ]
```

Comment:
Breaking change in `ts-jest`? Or just cleanup?

Reply:
Deprecated. It was throwing a warning.